test/integration/webhook/operatingsystemconfig/resources/crd-extensions.gardener.cloud_clusters.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.17.2
+  name: clusters.extensions.gardener.cloud
```

Comment:
All of these CRDs are now duplicated with the ones in `test/integration/controller/lifecycle/resources`. Can't we generate them only once and reference them then from all integration tests?

For example, see provider-aws integration tests. CRDs are generated only once under the example dir: https://github.com/gardener/gardener-extension-provider-aws/tree/v1.61.1/example. Later on, all integration tests reference them from there: https://github.com/gardener/gardener-extension-provider-aws/tree/v1.61.1/test/integration